components/scream/src/physics/p3/p3_constants.hpp

```diff
@@ -37,6 +37,7 @@ struct Constants
   static constexpr Scalar PIOV6       = Pi*SXTH;
```

Comment:
A general comment about p3_constants.hpp - we will eventually want to put most of these constants into a file shared by all of SCREAM or perhaps all of E3SM. p3_constants.hpp is ok for now, but we should get this more general file set up soon.